projects/packages/jetpack-mu-wpcom/.phpcs.dir.xml

```diff
@@ -23,4 +23,8 @@
 		</properties>
 	</rule>
 
+	<rule ref="MediaWiki.Usage.ForbiddenFunctions">
+		<exclude name="MediaWiki.Usage.ForbiddenFunctions.isset"/>
```

Comment:
Can we put this into `projects/packages/jetpack-mu-wpcom/src/features/newspack-blocks/synced-newspack-blocks/.phpcs.dir.xml` instead of excluding it for everything in the entire package?

Or even better, fix it upstream? 😀

Reply:
I'd put it into `synced-newspack-blocks/.phpcs.dir.xml` and we can fix it upstream in the future 🙂